src/os/windows/crashlog_win.cpp

```diff
@@ -206,11 +206,12 @@ static char *PrintModuleInfo(char *output, const char *last, HMODULE mod)
 /* virtual */ char *CrashLogWindows::LogModules(char *output, const char *last) const
 {
 	MakeCRCTable(AllocaM(uint32, 256));
-	BOOL (WINAPI *EnumProcessModules)(HANDLE, HMODULE*, DWORD, LPDWORD);
+	DllLoader psapi(L"psapi.dll");
+	BOOL (WINAPI *EnumProcessModules)(HANDLE, HMODULE*, DWORD, LPDWORD) = psapi.GetProcAddress("EnumProcessModules");
 
 	output += seprintf(output, last, "Module information:\n");
 
-	if (LoadLibraryList((Function*)&EnumProcessModules, "psapi.dll\0EnumProcessModules\0\0")) {
+	if (psapi.Success()) {
```

Comment:
If we're happy to make XP our minimum version, then EnumProcessModules can be statically linked.

Reply:
Minimum version is already XP. I'll check which dynamic loading can be removed.

Reply:
Hmm https://docs.microsoft.com/en-us/windows/win32/api/psapi/nf-psapi-enumprocessmodules says minimum XP, but desktop only. Is the windows store version still a desktop app ?

Reply:
Yes - although I did experiment with a UWP app, I don't think it's necessary now that desktop apps can go on the store.